db/fixtures/electoral_calculus_constituencies_tsv.rb

```diff
@@ -0,0 +1,86 @@
+require "csv"
+require_relative "constituencies_original_with_ons_csv"
+
+class ElectoralCalculusConstituenciesTsv
+  include Enumerable
+
+  TSV_FILE_NAME = "./db/fixtures/electoral_calculus_constituencies.tsv"
+
+  def initialize
+    @parties_by_party_code = {
+      "con"    => Party.find_by(name: "Conservatives"),
```

Comment:
This is kind of duplicating hardcoded data already in `seeds.rb`; can we factor them both out into `db/seeds/parties.rb` which just defines a `PARTIES` `Hash` mapping the abbreviation to `[full_name, colour]` or something like that?  However I'm OK with doing that in a follow-up PR.

Reply:
I spent a good while trying to figure out a neat solution to that, and that's what I came up with. Thing is, the mapping "con"   <=> "Conservatives" (which is essentially what's going on here) is a concern purely of this file and of nothing else. It has no purpose outside this file. The name "Conservatives" does ... but not the short code "con" which is just a convenience in the code that picks out the data from the columns of the csv/tsv. 

I'd say the refactoring could go 2 ways.
1) If we have any other use for the short code, put it in the database table.
2) if we decide it's ungainly, we might replace the code further down with
```
{
"Conservatives" => data[9],  ...
```
or whatever and eliminate the shortcode entirely.

Reply:
> Thing is, the mapping "con" <=> "Conservatives" (which is essentially what's going on here) is a concern purely of this file and of nothing else. It has no purpose outside this file.

Not sure I follow - the exact same mapping is needed in `seeds.rb` below.

Reply:
But appeasing my OCD tendencies for following DRY is not nearly as important as other stuff on the list, so probably just forget about it for now ;-)

Reply:
> the exact same mapping is needed in seeds.rb below
In that case my intent to delete it (because I know it's not used) was forgotten.

I will fix it now.

Reply:
OK thanks!